### PR TITLE
Add a new tutorial for GKE on the updated quickstart

### PR DESCRIPTION
**Pull request by joewhite101**

### CODE REVIEW

File: vuepress/docs/.vuepress/next.js [hunk 1]
```diff
@@ -274,7 +274,7 @@ module.exports = {
                     },
                     {
                         title: 'Installation on Google Kubernetes Engine (GKE)',
-                        path: path  + 'devops/installation/google-cloud-platform/',
+                        path: path  + 'devops/installation/google-cloud-platform/gke-quickstart.md',
```
Comment (nshaw) [inline vuepress/docs/.vuepress/next.js:277]: Should be gke-install.md

File: vuepress/docs/next/tutorials/devops/installation/google-cloud-platform/gke-install.md [hunk 2]
```diff
@@ -0,0 +1,237 @@
+1. Login to your Google Cloud account: <https://cloud.google.com/>
```
Comment (nshaw) [inline vuepress/docs/next/tutorials/devops/installation/google-cloud-platform/gke-install.md:1]: I think we lost some content on the new version. Usually there's an Overview, Prerequisites, etc. That could probably be lifted from the previous version.

File: vuepress/docs/next/tutorials/devops/installation/google-cloud-platform/gke-install.md [hunk 3]
```diff
@@ -0,0 +1,237 @@
...
+
+
+## Deploy Your Entando Application
+You can now deploy your Entando applications to GKE.
+
+### Setup and Deploy
+1. Download and unpack the entando-helm-quickstart:
+
+```
+curl -sfL https://github.com/entando-k8s/entando-helm-quickstart/archive/v6.3.2.tar.gz | tar xvz
```
Comment (nshaw) [inline vuepress/docs/next/tutorials/devops/installation/google-cloud-platform/gke-install.md:123]: This tag doesn't exist yet, right?

File: vuepress/docs/next/tutorials/devops/installation/google-cloud-platform/gke-install.md [hunk 4]
```diff
@@ -0,0 +1,237 @@
...
+
+In order to scale an Entando Application across multiple nodes you must provide a storage class that supports
+a `ReadWriteMany` access policy. There are many ways to accomplish this including using dedicated storage providers
+like GlusterFS or others.
+
+The instructions below provide an example of configuring using the GCP Cloud Filestore to provide the clustered storage
+but if you have an existing enterprise clustered file solution and you can expose it as a StorageClass it is recommended to
+use and test that configuration.
+
+::: note
```
Comment (nshaw) [inline vuepress/docs/next/tutorials/devops/installation/google-cloud-platform/gke-install.md:186]: Should be ::: tip to display correctly.

**Review by nshaw — CHANGES_REQUESTED** [review]

File: vuepress/docs/.vuepress/next.js (hunk 1 above)
Comment (joewhite101) [reply vuepress/docs/.vuepress/next.js:277]: fixed.

File: vuepress/docs/next/tutorials/devops/installation/google-cloud-platform/gke-install.md (hunk 2 above)
Comment (joewhite101) [reply vuepress/docs/next/tutorials/devops/installation/google-cloud-platform/gke-install.md:1]: added

File: vuepress/docs/next/tutorials/devops/installation/google-cloud-platform/gke-install.md (hunk 4 above)
Comment (joewhite101) [reply vuepress/docs/next/tutorials/devops/installation/google-cloud-platform/gke-install.md:186]: fixed

**Review by nshaw — APPROVED** [review]

**Pull request merged**